src/components/discardChanges.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="container">
+        <header class="header">
+            <div class="header-controls">
+                <button 
+                    @click="discardChanges" 
+                    class="btn-secondary btn" 
+                    :class="{'disabled': !wasSomethingChanged}">
+                    Discard
+                </button>
+            </div>
+        </header>
+    </div>
+</template>
+
+<script setup>
+import { ref, onMounted } from 'vue';
+
+// Референси для стейту
+const formula = ref({});
+const formulaBackup = ref({});
+const wasSomethingChanged = ref(false);
+
+// Метод для отримання даних із localStorage
+const loadFromLocalStorage = () => {
+    const storedData = localStorage.getItem('formulaBackup'); // Отримуємо дані з localStorage
```

Comment:
I am not sure we have "formulaBackup" object in localStorage
Also, it's better to use the `getState` from the already-created function